tensorrt_llm/_torch/models/modeling_nemotron_nano.py

```
@@ -234,7 +466,23 @@ def forward(
         modality_types = [
             multimodal_data["modality_type"] for multimodal_data in multimodal_data_lst
         ]
-        # Batch data.
+
+        has_dynamic = any(
+            "imgs_sizes" in multimodal_data.get(modality_type, {})
+            for modality_type, multimodal_data in zip(modality_types, multimodal_data_lst)
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)